sim:
  model_path: /assets/hector_v2/mvmc.xml
  headless: true
  viewer_fps: 30

robot:
  class_entry: hector_mujoco.src.robot_base.RobotBase
  verbose: true
  command2actuator:
    0: l_hip_yaw
    1: l_hip_roll
    2: l_hip_pitch
    3: l_knee
    4: l_ankle
    5: r_hip_yaw
    6: r_hip_roll
    7: r_hip_pitch
    8: r_knee
    9: r_ankle
    10: l_shoulder_yaw
    11: l_shoulder_pitch
    12: l_shoulder_roll
    13: l_elbow
    14: r_shoulder_yaw
    15: r_shoulder_pitch
    16: r_shoulder_roll
    17: r_elbow
  state2actuator:
    0: l_hip_yaw
    1: l_hip_roll
    2: l_hip_pitch
    3: l_knee
    4: l_ankle
    5: r_hip_yaw
    6: r_hip_roll
    7: r_hip_pitch
    8: r_knee
    9: r_ankle
    10: l_shoulder_yaw
    11: l_shoulder_pitch
    12: l_shoulder_roll
    13: l_elbow
    14: r_shoulder_yaw
    15: r_shoulder_pitch
    16: r_shoulder_roll
    17: r_elbow